tutorials/deutsch_jozsa/deutsch_jozsa.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<p>Consider also retrieving the results from the SDK and discussing the specific example we are using here.</p><p> </p><p>What are the results we get?</p><p> </p><p>What do they mean? Is the function balanced or constant?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Thanks, this is an insightful comment!